types/node/v11/test/repl.ts

```diff
@@ -1,54 +1,55 @@
-import { start, Recoverable } from "repl";
-import { Context } from "vm";
+// tslint:disable:space-before-function-paren
```

Comment:
Seems like this rule should be disabled for the whole repo at some point, as it clashes with `prettier`.